packages/vue-docgen-api/src/parse-template.ts

```diff
@@ -26,7 +26,11 @@ export default function parseTemplate(
 	if (tpl && tpl.content) {
 		const source =
 			tpl.attrs && tpl.attrs.lang === 'pug'
-				? pug.render(tpl.content.trim(), { ...pugOptions, filename: filePath })
+				? pug.render(tpl.content.trim(), {
+						...pugOptions,
+						doctype: 'html',
+						filename: filePath
```

Comment:
I believ it would be better as a default than as a force. Some people might want to use the pug compiler in a weird way and it's their choice ;) 
```suggestion
						doctype: 'html',
						...pugOptions,
						filename: filePath
```

what do you think ?